docs/staking.md

```diff
@@ -0,0 +1,48 @@
+
+Before nominating, you should always do your due diligence and research the credibility of the chosen validators. You can do so by checking their identity as well as historical information like era points, elected stake, rewards and slashes. At the beginning of Snakenet it might be difficult to find all this information - if you are in doubt about the choice of validators, approach us in Discord and we will share with you our community list of trusted validators.
+
+Another point to consider when choosing a validator is the *reward commission percentage*. This represents the proportion of the rewards which will be paid out to the validator for providing its services to nominators. The lowest commission percentage is not always the best - running a performant and available node has high operational costs which can only be covered sustainably by demanding a realistic reward commission.
+
+:::note
+
+We strongly recommend that you spread your stake by **nominating several validators**. This enhances the decentralization of the network and increases your chances of electing a trustworthy node in the set of active validators.
+
+Make sure that you do not nominate validators which are oversubscribed. Currently, there is a **limit of 64 nominations** for a single validator, after which it becomes oversubscribed. When the following era begins, an oversubscribed validator will only be elected using the maximum allowed number of nominations. If this occurs, the nominations with the highest stake take precedence, while the nominations with the lowest stake will be disregarded and will not earn any awards during the era.
+
+Nominating is a more accessible form of staking however it also bears risks. Validators which breach the rules of the network may be punished by slashing, resulting in a partial loss of funds for both the validator and its nominators. This is why it is important to only nominate reputable validator nodes.
+
+:::
+
+After you have nominated one or more validators, you will not start earning rewards immediately. Before this can happen, the validators in question must receive sufficient nominations and staked tokens to be elected in the set of active validators. Until then, your nomination will remain inactive (for more details see previous section).
```

Comment:
```suggestion
After you have nominated one or more validators, you will not start earning rewards immediately. Before this can happen, the validators in question must receive sufficient staked tokens to be elected in the set of active validators. Until then, your nomination will remain inactive (for more details see previous section).
```

Reply:
sufficient backing?